k4MarlinWrapper/k4MarlinWrapper/converters/EDM4hep2Lcio.h

```diff
@@ -0,0 +1,190 @@
+#ifndef K4MARLINWRAPPER_EDM4HEP2LCIO_H
+#define K4MARLINWRAPPER_EDM4HEP2LCIO_H
+
+
+// GAUDI
+#include "GaudiAlg/GaudiTool.h"
+
+// FWCore
+#include "k4FWCore/DataHandle.h"
+
+// EDM4hep
+#include "edm4hep/ParticleIDCollection.h"
+#include "edm4hep/ReconstructedParticle.h"
+#include "edm4hep/ReconstructedParticleCollection.h"
+#include "edm4hep/ReconstructedParticleData.h"
+#include "edm4hep/Track.h"
+#include "edm4hep/TrackCollection.h"
+#include "edm4hep/TrackerHit.h"
+#include "edm4hep/TrackerHitCollection.h"
+#include "edm4hep/SimTrackerHit.h"
+#include "edm4hep/SimTrackerHitCollection.h"
+#include "edm4hep/CalorimeterHit.h"
+#include "edm4hep/CalorimeterHitCollection.h"
+#include "edm4hep/RawCalorimeterHit.h"
+#include "edm4hep/RawCalorimeterHitCollection.h"
+#include "edm4hep/SimCalorimeterHit.h"
+#include "edm4hep/SimCalorimeterHitCollection.h"
+#include "edm4hep/TPCHit.h"
+#include "edm4hep/TPCHitCollection.h"
+#include "edm4hep/Cluster.h"
+#include "edm4hep/ClusterCollection.h"
+#include "edm4hep/Vertex.h"
+#include "edm4hep/VertexCollection.h"
+#include "edm4hep/MCParticle.h"
+#include "edm4hep/MCParticleCollection.h"
+
+// LCIO
+#include <LCEventWrapper.h>
```

Comment:
Better to change the include directories so this has to be prefixed by the folder
```suggestion
#include <k4MarlinWrapper/LCEventWrapper.h>
```

Reply:
I would suggest renaming `k4MarlinWrapper/k4MarlinWrapper` to `k4MarlinWrapper/include` and keep the includes as they are: putting the files names from inside the include folder. That way we have the `src` and `include` folders inside the k4MarlinWrapper folder and avoid having a triple nested k4MarlinWrapper path.

Reply:
I would like to have "k4MarlinWrapper" before, because if the header files are installed they don't end up in a global "include" folder but in "include/k4marlinwrapper" when doing a"view" type thing.